src/v/redpanda/application.cc

```diff
@@ -852,6 +865,16 @@ void application::hydrate_config(const po::variables_map& cfg) {
         config::node().rack.set_value(std::nullopt);
     }
 
+    // load ID overrides
+    if (!cfg["node-id-overrides"].empty()) {
+        // TODO(oren): maybe we only need these on shard 0?
```

Comment:
nit: remove TODO. I would keep setting it on all shards to ensure that `config::node()` is consistent across shards.

Reply:
done